packages/aws-cdk-lib/aws-autoscaling/README.md

```diff
@@ -288,6 +288,38 @@ autoScalingGroup.scaleOnSchedule('AllowDownscalingAtNight', {
 });
 ```
 
+### Instance Maintenance Policy
+
+The instance maintenance policy allows you to configure an instance maintenance policy for
+your Auto Scaling group to meet specific capacity requirements during events that cause
+instances to be replaced, such as an instance refresh or the health check process.
+
+For example, suppose you have an Auto Scaling group that has a small number of instances.
+You want to avoid the potential disruptions from terminating and then replacing an instance
+when health checks indicate an impaired instance. With an instance maintenance policy, you
+can make sure that Amazon EC2 Auto Scaling first launches a new instance and then waits for
+it to be fully ready before terminating the unhealthy instance.
+
+An instance maintenance policy also helps you minimize any potential disruptions in cases where
+multiple instances are replaced at the same time. You set the `maintenancePolicyMinHealthPercentage`
+and the `maintenancePolicyMaxHealthPercentage` for the policy, and your Auto Scaling group can only
+increase and decrease capacity within that minimum-maximum range when replacing instances.
+A larger range increases the number of instances that can be replaced at the same time.
+
+```ts
+declare const vpc: ec2.Vpc;
+
+new autoscaling.AutoScalingGroup(this, 'ASG', {
+  vpc,
+  instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.MICRO),
+  machineImage: ec2.MachineImage.latestAmazonLinux2(),
+  maintenancePolicyMaxHealthPercentage: 200,
+  maintenancePolicyMinHealthPercentage: 100,
```

Comment:
I'm not necessarily against this, but what was the reasoning against just `minHealthPercentage` and `maxHealthPercentage`? I think i prefer that unless it would be confusing if minimum health could point to other things besides maintenance policy.

Reply:
Yes, I was not sure at first, but I think I will do that.

Reply:
Changed to `minHealthyPercentage` and `maxHealthPercentage` to match CloudFormation.